radeon-profile/dxorg.cpp

```diff
@@ -555,7 +555,7 @@ GPUPwmStruct dXorg::getPwmSpeed() {
 
     QFile f(hwmonAttributes.pwm1);
 
-    if (f.open(QIODevice::ReadOnly)) {
+    if (f.exists() && f.open(QIODevice::ReadOnly)) {
```

Comment:
Additional check is not needed, since `open()` will fail anyway if file doesn't exist.